files/en-us/web/css/anchor/index.md

```diff
@@ -12,17 +12,17 @@ The **`anchor()`** [CSS](/en-US/docs/Web/CSS) [function](/en-US/docs/Web/CSS/CSS
 ## Syntax
 
 ```css
-/* property: anchor(anchor-side) */
+/* side or percentage */
```

Comment:
Changed this to be more in line with other information syntax sections; lay terms rather than actual syntax.